supabase/tests/fixtures/privileged_metrics.txt

```diff
@@ -0,0 +1,924 @@
+node_memory_Mlocked_bytes{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db"} 4096
+# HELP node_filesystem_files Filesystem total file nodes.
+# TYPE node_filesystem_files gauge
+node_filesystem_files{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db",device="/dev/nvme0n1p2",fstype="ext4",mountpoint="/"} 648960
+node_filesystem_files{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db",device="/dev/nvme1n1",fstype="ext4",mountpoint="/data"} 524288
+# HELP node_filesystem_readonly Filesystem read-only status.
+# TYPE node_filesystem_readonly gauge
+node_filesystem_readonly{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db",device="/dev/nvme0n1p2",fstype="ext4",mountpoint="/"} 0
+node_filesystem_readonly{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db",device="/dev/nvme1n1",fstype="ext4",mountpoint="/data"} 0
+# HELP node_memory_Active_file_bytes Memory information field Active_file_bytes.
+# TYPE node_memory_Active_file_bytes gauge
+node_memory_Active_file_bytes{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db"} 8.4094976e+07
+# HELP node_memory_Slab_bytes Memory information field Slab_bytes.
+# TYPE node_memory_Slab_bytes gauge
+node_memory_Slab_bytes{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db"} 7.243776e+07
+# HELP node_vmstat_pgfault /proc/vmstat information field pgfault.
+# TYPE node_vmstat_pgfault untyped
+node_vmstat_pgfault{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db"} 2.18115e+06
+# HELP node_cpu_guest_seconds_total Seconds the CPUs spent in guests (VMs) for each mode.
+# TYPE node_cpu_guest_seconds_total counter
+node_cpu_guest_seconds_total{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db",cpu="0",mode="nice"} 0
+node_cpu_guest_seconds_total{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db",cpu="0",mode="user"} 0
+node_cpu_guest_seconds_total{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db",cpu="1",mode="nice"} 0
+node_cpu_guest_seconds_total{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="db",cpu="1",mode="user"} 0
+# HELP pg_stat_database_xact_commit_total Transactions committed
+# TYPE pg_stat_database_xact_commit_total counter
+pg_stat_database_xact_commit_total{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="postgresql",server="localhost:5432"} 2472
+# HELP physical_replication_lag_is_wal_replay_paused Check if WAL replay has been paused
+# TYPE physical_replication_lag_is_wal_replay_paused gauge
+physical_replication_lag_is_wal_replay_paused{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="postgresql",server="localhost:5432"} 0
+# HELP postgres_exporter_config_last_reload_success_timestamp_seconds Timestamp of the last successful configuration reload.
+# TYPE postgres_exporter_config_last_reload_success_timestamp_seconds gauge
+postgres_exporter_config_last_reload_success_timestamp_seconds{supabase_project_ref="xdgggqomhgiilfjtepts",service_type="postgresql"} 0
```

Comment:
Do you know if this was manually created? Seems like this metric also references a timestamp?

Reply:
Discussed separately - this was not manually edited. Even though it references a timestamp, we can't use time_elapsed here since it regularly reports 0's.